lib/index.js

```diff
@@ -113,7 +113,7 @@ class ServerlessFargateTasks {
         } else {
           var datadog_agent_definition = {
             'Name': `${this.service.service}-${this.stage}-datadog-agent`,
-            'Image': "public.ecr.aws/datadog/agent:latest",
+            'Image': "public.ecr.aws/datadog/agent:7.50.3",
```

Comment:
Suggestion, we could make this an option with a default of `latest`. See below how the `options.datadog.cpu || 10` allows overriding the default. 

Then each individual repo could control which version it's using without making this plugin stuck on 7.50.3

Reply:
Thanks Ryan! Would this be correct?

'Image': "public.ecr.aws/datadog/agent:7.50.3" || 'Image': "public.ecr.aws/datadog/agent:latest",

Reply:
I think you could do it like this using template strings: 
`'Image': `public.ecr.aws/datadog/agent:${options.datadog.image_tag || "latest"}`,`

Reply:
`options.datadog` is flexible and adding something to that would let you do this in the ETL repos: 
```yaml
...
fargate:
    datadog:
      ssm_api_key: /Keys/Datadog/ApiKey
      image_tag: 7.50.3
   ...
```

Reply:
Ok, it means changing all repo' serverless.yml using this plugin to add the new variable?

Reply:
Yeah exactly. Each repo needs to be updated anyway to install the newer version of this plugin. I think there are only a handful of repos that use this plugin

Reply:
@rwhitten577 can you approve this please when you get a sec? Subsequently I'll follow in with a PR for internal-ETL, to begin with.

Reply:
Let's test this out locally first. You can install this package into internal-etl by running `npm install --save -D ../serverless-fargate-tasks/` from the internal-etl project, and adjusting the path as needed for your local files. Then you can run the deploy script and should see the datadog version respected in the ECS task definition